packages/web-frontend/src/screens/LandingPage/useCustomLandingPages.js

```diff
@@ -2,6 +2,7 @@
  * Tupaia
  *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
+import { useEffect } from 'react';
```

Comment:
Does this end up getting used?

Reply:
No. I initially had the useEffect in the hook but moved it out to the main page to make it easier to find.